src/adplug/adplug-xmms.cc

```diff
@@ -171,15 +192,30 @@ bool AdPlugXMMS::play (const char * filename, VFSFile & fd)
   dbg_printf ("open, ");
   open_audio (bit16 ? FORMAT_16 : FORMAT_8, freq, stereo ? 2 : 1);
 
-  CEmuopl opl (freq, bit16, stereo);
+  Copl *opl = nullptr;
```

Comment:
I would prefer to use a `SmartPtr` here.

Reply:
That's easy enough. When I get opportunity I'll do that.

Reply:
Changed in commit 3d8642b.